meson.build

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+project('qrtr',
+        'c',
+        license : [ 'BSD-3-Clause'],
+        meson_version : '>= 0.64.0', # for feature user options
+        default_options : [
+        #     'warning_level=2',	# sets -Wextra
+        'buildtype=release',
+        ])
+
+# Set advanced compiler flags
+compiler = meson.get_compiler('c')
+
+prefix = get_option('prefix')
+systemd = dependency('systemd')
+systemd_system_unit_dir = systemd.get_variable(
+        pkgconfig : 'systemdsystemunitdir',
+        pkgconfig_define: ['prefix', prefix])
```

Comment:
On non-usr-merge systems the systemd units usually went to `/lib/systemd`, while the prefix was `/usr`. With Makefiles it was easy to override using the command line. Can we retaing the possibility to override systemd units location please?